src/bootstrap/config.rs

```diff
@@ -1112,7 +1112,8 @@ impl Config {
 
         // Set flags.
         config.paths = std::mem::take(&mut flags.paths);
-        config.exclude = flags.exclude;
+        config.skip = flags.skip;
+        config.skip = flags.exclude;
```

Comment:
This kinda suspicious: silently overwriting one flag with other one.

Reply:
This is true if there some warn about this behavior, otherwise this is bug, IMHO.

Reply:
Wouldn't it make sense to merge (`extend`) the two instead of overwriting `flags.skip`?

Reply:
absolutely, extend is the right way to do it

Reply:
Do you mind if I stop bors, then?

Reply:
It's safer to r- r+, as approved commit differs from last one? bors kind glitchy around that things.